vector/src/main/res/menu/menu_devtools.xml

```diff
@@ -17,6 +18,7 @@
         tools:visible="true"
         app:showAsAction="ifRoom"
         android:icon="@drawable/ic_send"
+        app:iconTint="?vctr_content_secondary"
```

Comment:
I can never remember if it's works fine on API21, but I guess for a debug screen this is fine.